org.omg.sysml/src/org/omg/sysml/util/traversal/facade/impl/JsonElementProcessingFacade.java

```diff
@@ -1,292 +1,294 @@
+	 */
+	public boolean isIncludeDerived() {
+		return this.isIncludeDerived;
+	}
+	
+	/**
+	 * Create an Identified object with the given identifier.
+	 * 
+	 * @param 	identifier			the UUID of the object being identified
+	 * @param isLibraryElement		set to true if the element is part of the standard library
+	 * @return	an Identified object with the given identifier
+	 */
+	protected static Map<String, UUID> identified(UUID identifier, String type, boolean isLibraryElement) {
+		return new APIModel.LocalReference(identifier, type, isLibraryElement);
+	}
+	
+	/**
+	 * Create an Identified object containing the identifier of the given element.
+	 * 
+	 * @param 	element				the Element to be identified
+	 * @return	an Identified object for the given Element (or null if the input is null).
+	 */
+	protected Map<String, UUID> getIdentified(Element element) {
+		return element == null? null: identified(UUID.fromString(element.getElementId()), element.eClass().getName(), element.isLibraryElement());
+	}
+	
+	/**
+	 * Create a list of Identified objects corresponding to the identifiers of a given list of Elements.
+	 * 
+	 * @param 	elements			the Elements being identified
+	 * @return	a list of Identified objects with the identifiers of the given elements
+	 */
+	protected List<Map<String, UUID>> getIdentified(List<Element> elements) {
+		return elements.stream().
+				map(this::getIdentified).
+				filter(id->id != null).
+				toList();
+	}
+
+	/**
+	 * Create an (API)Element for the given model (EMF)Element including the values of all its non-derived  
+	 * attributes (unless it is a library model element, in which case only its non-referential attribute values 
+	 * are included). If isIncludeDerived is true, also include derived attributes. The ID for the DataVersion 
+	 * uses the identifier from the model Element.
+	 * 
+	 * @param 	element				the source model Element as it is represented in Ecore
+	 * @return	API representation of the given Element
+	 */
+	@SuppressWarnings("unchecked")
+	protected org.omg.sysml.model.Element createApiModelElement(Element element) {
+		org.omg.sysml.model.Element apiElement = new org.omg.sysml.model.Element();
+		EClass eClass = element.eClass();
+		apiElement.put("@type", eClass.getName());
+		apiElement.put("@id", element.getElementId());
+		boolean isLibraryElement = ElementUtil.isStandardLibraryElement(element);
+		for (EStructuralFeature feature: eClass.getEAllStructuralFeatures()) {
+			String className = eClass.getName();
+			String featureName = feature.getName();
+			//always add the importedElement derived field. This field is used for a workaround that addresses
+			//unstable library membership UUIDs in the EMFModelRefresher.
+			if ((this.isIncludeDerived() || !feature.isDerived() || "importedElement".equals(feature.getName())) && 
+					// Skip implementation-specific features.
+					!(/*"Feature".equals(className) && */"isNonunique".equals(featureName) || 

```

Comment:
Either remove the commented out section, or add a comment stating why is it necessary to keep it in the codebase.